### PR TITLE
Feature/cmake

### PR DESCRIPTION
**Pull request by gjbex**
Added GCC version for ncurses and CMake

### CODE REVIEW

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = 'ncurses'
```
Comment (wpoely86) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.9.2.eb]: please add
```
easyblock = 'ConfigureMake'
```

**Comment by wpoely86** [comment]
ok to test

File: easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.9.2.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/n/ncurses/ncurses-5.9-GCC-4.9.2.eb]: no, there's an easyblock for ncurses (otherwise the tests would fail)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9b0adcb15e02f9d9a800 for a full test report.

**Comment by boegel** [comment]
looks great, works fine, so going in, thanks @gjbex

**Pull request merged**